src/ratingservice/.gcloudignore

```diff
@@ -0,0 +1,19 @@
+# This file specifies files that are *not* uploaded to Google Cloud Platform
```

Comment:
you might need to include the license header there

Reply:
I added it.